.github/workflows/rust.yml

```diff
@@ -126,4 +126,4 @@ jobs:
   #       uses: actions/upload-artifact@v2
   #       with:
   #         name: code-coverage-report
-  #         path: cobertura.xml
+  #         path: cobertura.xml
```

Comment:
Do not commit this change, please config your editor to add a blank line at end of file